File: Python/Slice/remove_element_from_slice_at_index.py
# Remove element from the slice at a certain index.
def remove_element_from_slice_at_index(provided_slice, provided_index):
    provided_slice.pop(provided_index)
    return provided_slice


def main():
    # Make a slice using a random set of data.
    randomSlice = ["c", "a", "b"]
    # Remove element from the slice at a certain index.
    print(remove_element_from_slice_at_index(randomSlice, 1))


main()

File: TypeScript/Slice/remove-element-from-slice-at-index.ts
function main(): void {
    // Create a slice of strings
    var randomSlice: string[] = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j"]
    // Remove element from slice at index
    console.log(removeElementFromSliceAtIndex(randomSlice, 3))
}

main()

// Remove element from slice at index
function removeElementFromSliceAtIndex(originalSlice: string[], index: number): string[] {
    return originalSlice.slice(0, index).concat(originalSlice.slice(index + 1))
}